### DISCUSSION TITLE
Where is `shellHook` placed?

### DISCUSSION THREAD

**Discussion by peterbecich** [srid/haskell-flake#132] [Q&A] [answered]
I am trying to apply this quick fix: https://github.com/mozilla/nixpkgs-mozilla/issues/240#issuecomment-682285166
```
shellHook = ''
  export LD_LIBRARY_PATH=${zlib.out}/lib:$LD_LIBRARY_PATH
'';
```

It may be the solution to this Cabal error:
```
libz.so.1: cannot open shared object file: No such file or directory
```

`zlib` is provided, already:
```
tools = hp: {
  inherit (pkgs)
    zlib;
};
```

Where would `shellHook` go in a `haskell-flake`? Thank you

**Comment by srid** [comment, marked as answer]
It's [`mkShellArgs.shellHook`](https://flake.parts/options/haskell-flake.html#opt-perSystem.haskellProjects._name_.devShell.mkShellArgs).
